web_developer_mode_optional/static/description/icon.svg

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
@jgrandguillaume, @gurneyalex : Then, what do I proceed with OCA modules icons?
I think any module could have its own icon. In that case, for any future modification, it should include source (in this case an .svg file), even if it makes a commit less readable.
If no particular icon is used, then default to OCA .png-only icon, as it is on templates.